src/diagnostic/diagnostic_model_view.hpp

```diff
@@ -185,7 +216,9 @@ class BaseModelView : public IModelView
     };
 
     std::vector<MTAlgo> MTAlgos;
-    TensorFieldT sumTensor_{"PHARE_sumTensor", core::HybridQuantity::Tensor::M};
+    Field tmpField_{"PHARE_sumField", core::HybridQuantity::Scalar::rho};
+    VecField tmpVec_{"PHARE_sumVec", core::HybridQuantity::Vector::V};
+    TensorFieldT tmpTensor_{"PHARE_sumTensor", core::HybridQuantity::Tensor::M};
```

Comment:
during PGL these were only doing a sum, but perhaps they should be renamed generally, to `PHARE_tmpX` or similar